.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Happy to remove, the reason I added was as it can be super useful for people coming to the repo to pick up easily and F5 in VSCode. As the file isn't .go it will be stripped out by dep when the SDK is imported so felt like a reasonable addition but can see the other side that it's only useful to those using VSCode so happy to remove. 

(We used quite a bit to capture linting settings, common tasks and debugging steps in one place - for example: https://github.com/lawrencegripper/ion/tree/master/.vscode )

Reply:
Done